chat-commands.js

```diff
@@ -2274,9 +2274,11 @@ exports.commands = {
 			target = targets[1].trim();
 			roomId = toId(targets[0]) || room.id;
 		}
+		let targetRoom = Rooms.get(roomId);
+		let addModlogLinks = Config.modloglink && (!hideIps || (Config.roomauthModlogLink && targetRoom && !targetRoom.isPrivate));
```

Comment:
I don't think it's commonly enough used by other servers that they wouldn't want to fork it anyway. Just treat it as always true.